### DISCUSSION TITLE
Can SOFA do steady state calculation ?

### DISCUSSION THREAD

**Discussion by SustechShenCong** [sofa-framework/sofa#4968] [SofaPython3] [answered]
Does SOFA can only perform transient calculations ? How to perform steady state calculations ?

**Comment by bakpaul** [comment, marked as answer]
Hello, if by steady state calculation you mean static solving (find the equilibrium of the système without passing through the dynamics) the yes. You'll need to use the StaticSolver ODE.

**Reply by SustechShenCong** [reply]
Your reply is so so helpful, thank you very much !
